<blockquote>
<p>The team has achieved significant research progress this year in four distinct areas: (i) phylogenetic analysis—studying the relationships between genomics and disease using advanced data science methods, (ii) ecological modelling—contrasting geographical and genomic differences within endangered species and (iii) live science publishing—creating a new way of publishing science experiments embedded within traditional research articles that are now getting traction with other researchers and (iv) AI tools that can validate or refute scientific claim--for example: "Covid vaccines cause sterility in males". In each of these areas we have developed new methods and related software that is being shared openly with the science community.</p>
<p>Collaborations within our team are now leading to additional grant funding in the areas of AI and genomics, and we have strengthened our collaborations with other research partners based in Aotearoa (notably ESR and Genomics Aotearoa with plans to extend to the national RNA Platform once it is underway).</p>
<p>We sponsored a significant national outreach &amp; training event (ResBaz 2022: https://resbaz.auckland.ac.nz/sessions/) enabling several hundred applied researchers throughout the country to learn new data science skills that they can apply in their own research. We are also now training a future generation of NZ data scientists via our PhD and post-doctoral scholarships and via related AI Carpentry work.</p>
<p>Taken together, this progress is producing the new methods, code, collaborations and workforce that will ensure Aotearoa can take better advantage of data science in its future research endeavours in key industries such as genomics and AI and branches of government such as public health and debate on science and policy.</p>
<p>For more information, contact Prof. Mark Gahegan, Professor of Computer Science at the University of Auckland, email: m.gahegan@auckland.ac.nz</p>
</blockquote>
<blockquote>
<p>Section 3: Progress and Key Achievements</p>
<p>Progress and Key Achievements</p>
</blockquote>
<blockquote>
<p><strong>Strategic intent</strong></p>
<p>Our overall strategic direction is to improve the tools and methods available to data scientists to enable them to more easily gain important insights, validate their work and share their findings. Two specific strategic themes of (i) Live Data Science and (ii) Transparent Data Science are presented below, along with key progress over the past year.</p>
<p><strong>Live Data Science—closing the gap between doing science and publishing science</strong></p>
<p><strong>The methods we use for sharing our science findings is woefully out of date and inefficient. In large part they have not changed in 350 years. Our aim here is to enable researchers to create an accurate, complete, verifiable and therefore trustworthy record of their research. This will lead to science that is more efficient (because it will be easier to reuse and repurpose published research) and more effective (because it can be verified, used more widely, shared more completely.</strong></p>
<p>Over the past year, the LivePublication project has significantly matured, gaining engagement and interest from various stakeholders. We have been fleshing out our <strong>Live data Science</strong> infrastructure, focussing for now on how to bring publications and report writing more in line with modern eScience toolchains and workflows. in 2022 we developed the first prototype of a live science pipeline that fully links live data ingestion through computational workflows to dynamic publication—i.e. a publication that updates itself as the data it consumes is either updated or improved. Starting in early in 2023, we began to collaborate with Prof. Ian Foster's supercomputing team at Argonne National Labs, Chicago, USA on expanding this project to use the Globus data infrastructure developed at Argonne (https://www.globus.org/) and used extensively throughout the world, along with their Gladier science workflow technology. We have also re-engineered the internal interfaces to describe research artefacts and outcomes using the Research Object Crate technology (https://www.researchobject.org/ro-crate/). This has caught the attention of some key data scientists in the UK (the Data Science And AI Institute at U. Manchester led by Prof. Carole Goble) who are also keen to work with us.</p>
</blockquote>
<blockquote>
<p>LivePublication’s tools, available at <u>LivePublication on Github</u>, are now operationalizing the first fully automated and live science research publication providing a solid base for future development and experimentation. A comparative case study for language identification models has been developed to showcase LivePublication’s capabilities, integrating GitHub for versioning and Quarto for rendering publications with dynamic elements. It can be explored here: <u>LivePublication Case Study</u>. With the operalization of LivePublication, a new case study is underway to highlight the significance of integration live data source within reactive articles. Further translational case studies are planned for later this year. Our latest publication on this research has been accepted for publication and presentation at the 19th IEEE International eScience Conference later this year (A. Ellerm, M. Gahegan, and B. Adams (2023) <em>"LivePublication: The science workflow creates and updates the publication</em>."</p>
<p>We recently completed a workshop with Stencila, a New Zealand start-up focused on interactive and collaborative scientific publishing (https://stencila.io/). The next version of LivePublication will utilise Stencila's technology to describe, mark up and share science publications and we will be working closely with Stencila into the future.</p>
<p><strong>Transparent Data Science—increasing trust in scientific claims</strong></p>
<p><strong>There is a crisis of public trust in science and in scientists. Alternative 'facts' can be damaging to public discourse if they are not grounded in any evidence, and if they cannot be easily differentiated from established truths. We urgently need tools that can test the validity of scientific claims, to enable policy discourse that is based on evidence and not hearsay.</strong></p>
<p>Learning how to verify the claims in scientific papers, science press releases or even on social media is a difficult task for both artificial intelligence (AI) systems and humans. However, this task is crucial because</p>
</blockquote>
<blockquote>
<p>learning to separate verified facts from speculation or falsehoods has important consequences. Success at this task can help the reader understand scientific topics and promote science. Conversely, failure at this task leads to the spread of misinformation and exaggeration, which can cause distortion in scientific communication and undermine public confidence in science.</p>
<p>Training machine learning models to successfully perform scientific fact-checking tasks is has to this point been challenging due to the expertise bottleneck that limits the availability of appropriate training datasets. In this task, models use textual evidence to confirm scientific claims, which requires data that can be used to train the models—i.e. extensive domain-expert annotation. Consequently, the number of existing scientific-fact-checking datasets and the sizes of those datasets are limited. However, these limitations do not apply to multiple-choice question datasets because of the necessity of domain exams in the modern education system.</p>
<p>Our work over the past year has overcome this bottleneck, allowing us to assess the validity of science claims against all of the published scientific literature, without the need for targetting just one narrow domain of science, and without extensive training beforehand. This is a significant breakthrough, attained in part because of the successes of Large Language Models used in Deep Learning.</p>
<p>Over the past year, we've developed a scientific reading comprehension system, enhancing transparency and trustworthiness in scientific claims. This allows a user to test any science claim (e.g. Covid vaccines cause sterility in males) against the published science literature. Each claim (and its opposite) is given a compound score based on how much published science supports the claim, and the various ranking metrics used to evaluate science (such as journal standing and number of citations). The most pertinent research articles are summarised for the user in the light of their specific question, so they can more easily see why a particular argument might be true, or not.</p>
<p>We have submitted a research paper to a top NLP conference [1], presented at another prestigious NLP conference [2], and published our datasets, all in line with promoting Open Science principles. In the coming year, we'll focus on achieving excellence by refining our system using cutting-edge data science and deep learning techniques. The impact will extend to researchers and the public, preventing the spread of misinformation, streamlining research processes, and advancing data science through interdisciplinary collaboration.</p>
</blockquote>
<blockquote>
<p>[1] Tan, Neset, et al. "Beyond Fact Checking: Faithful Reasoning over Scientific Claims." EMNLP23.</p>
</blockquote>
<blockquote>
<p>[2] Tan, Neset, et al. "Multi2Claim: Generating Scientific Claims from Multi-Choice Questions for Scientific Fact-Checking." Proceedings of the 17th Conference of the European Chapter of the Association for Computational Linguistics, 2023.</p>
</blockquote>
<blockquote>
<p><strong>Excellence</strong></p>
<p><strong>We provide 3 examples of excellence for the past year, in (i) Meta-Data Science, (ii) Genome Simulations and (iii)expanding our team with new alliances and additional funding</strong></p>
<p><strong>Example 1: Meta-data Science</strong></p>
<p>In meta data science, we advanced three lines of research this year: (1) the discovery of data dependencies, (2) the design of data spaces, and (3) the management of data integrity.</p>
<p>Our main contribution towards (1) is the first complete approach towards the discovery of meaningful functional dependencies (FDs), published in [1] below, which is made up from the set of true positives arising from the output of algorithms that return all FDs that hold on a given data set and the set of false negatives. The publication does not only advance the state-of-the-art on FD discovery algorithms by a new hybrid approach, but also includes first methods on ranking the outputs to separate true from false positives, and on ranking FDs not included in the output to separate false from true negatives.</p>
<p>In addressing (2), we have published a first approach [3] towards the design of SQL table schemata that is not only based on FDs but also cardinality constraints, which improves state-of-the-art by the ability of quantifying the achievements of classical relational database design by measuring update inefficiency</p>
</blockquote>
<blockquote>
<p>and join efficiency already during the logical design of SQL table schemata, where no data is present yet. The measure is shown experimentally to be a strong predictor at the operational level, ones data tables have become available. Furthermore, we have parameterized classical relational Boyce-Codd Normal Form (BCNF) by the number of minimal keys [4], making it possible to break ties between different schemata in BCNF. As the number of minimal keys quantifies the trade-off between data integrity and data access variety, we have optimized classical relational database design algorithms towards either update efficiency or query efficiency. This has been validated experimentally on benchmark data.</p>
<p>Finally, we have extended state-of-the-art on data integrity management by automating entity integrity management using key sets rather than single keys [5], introducing uniqueness constraints to object stores [6], and utilizing classical keys, functional and multivalued dependencies in data spaces where the dimensions of data volume, variety, and veracity are accommodated [7].</p>
<p><strong>Discovery of data dependencies:</strong></p>
</blockquote>
<blockquote>
<p>[1] WEI, Z., LINK, S.: `Towards the Efficient Discovery of Meaningful Functional Dependencies’, Information Systems, Volume 116:102224, <u>https://doi.org/10.1016/j.is.2023.102224</u>, <strong>CORE: A*</strong>, 2023.</p>
</blockquote>
<blockquote>
<p>[2] HANNULA, M.; ZHANG, Z.; SONG B.K.; LINK, S.: `Discovery of Cross Joins (Extended Abstract)’, In:</p>
</blockquote>
<blockquote>
<p>Proceedings of the 39<sup>th</sup> International Conference on Data Engineering - ICDE, pages: 3853-3854,</p>
<p><u>https://doi.org/10.1109/ICDE55515.2023.00353</u>, <strong>CORE: A*</strong>, 2023.</p>
<p><strong>Design of data spaces:</strong></p>
</blockquote>
<blockquote>
<p>[3] LINK, S.; KOEHLER, H.; GANDHI, A., HARTMANN, S., THALHEIM, B.: `Cardinality Constraints and</p>
</blockquote>
<blockquote>
<p>Functional Dependencies in SQL: Taming data redundancy in logical database design’, Information</p>
<p>Systems, Volume 115:102208, <u>https://doi.org/10.1016/j.is.2023.102208</u>, <strong>CORE: A*</strong>, 2023.</p>
<p>[4] ZHANG, Z.; CHEN, W.; LINK, S.: `Composite Object Normal Forms: Parameterizing Boyce-Codd Normal Form by the Number of Minimal Keys’, Proceedings of the ACM International Conference on Management of Data (PACMMOD), Volume 1(1):13.1-13.25, <u>https://doi.org/10.1145/3588693</u>, <strong>CORE: A*</strong>, 2023.</p>
</blockquote>
<blockquote>
<p><strong>Data integrity management:</strong></p>
</blockquote>
<blockquote>
<p>[5] HANNULA, M., LI, X., LINK, S.: `Controlling Entity Integrity with Key Sets’, Journal of Computer and</p>
</blockquote>
<blockquote>
<p>System Science, Volume 136:195-219, <u>https://doi.org/10.1016/j.jcss.2023.04.004</u>, <strong>CORE: A*</strong>, 2023.</p>
<p>[6] SKAVANTZOS, P.; LECK, U.; ZHAO, K.; LINK, S.: `Uniqueness Constraints for Object Stores’, ACM Journal of Data and Information Quality, Just accepted, <u>https://doi.org/10.1145/3581758</u>, 2023.</p>
</blockquote>
<blockquote>
<p>[7] LINK, S.: `Entity Integrity Management Under Data Volume, Variety, and Veracity’, Knowledge and Information Systems, Volume 65:2895-2934, <u>https://doi.org/10.1007/s10115-022-01814-1</u>, 2023.</p>
</blockquote>
<blockquote>
<p><u>Conference talks:</u></p>
</blockquote>
<blockquote>
<p>[t1] ZHANG, Z.; CHEN, W.; LINK, S.: `Composite Object Normal Forms: Parameterizing Boyce-Codd</p>
</blockquote>
<blockquote>
<p>Normal Form by the Number of Minimal Keys’, SIGMOD 2023, <strong>CORE: A*</strong>, 2023.</p>
<p><u>https://prezi.com/view/nT9POESONgmWqXxivO8V/</u></p>
<p>[t2] HANNULA, M.; ZHANG, Z.; SONG B.K.; LINK, S.: `Discovery of Cross Joins (Extended Abstract)’,</p>
</blockquote>
<blockquote>
<p>ICDE 2023, <strong>CORE: A*</strong>, 2023.</p>
<p><strong>7 of these 9 outcomes are CORE-ranked as A* journals or conference, with highly competitive review processes Our peers clearly rate our work!</strong></p>
</blockquote>
<blockquote>
<p><strong>Example 2: Genome Simulation</strong></p>
<p>Postdoctoral researcher Alistair Lamont has been working closely with Phillip Wilcox and Mik Black to develop a new, efficient, genome simulator that can incorporate key geographic and structural parameters. This is the next step in the design of tools appropriate for genomic analysis and prediction within specific culture communities, particularly iwi not well-served by existing tools with generic assumptions and potential euro-centric data biases. The simulator will also be critical for the design and implementation (in collaboration with Bryant and Gardner) of new analytical tools.</p>
<p>Two talented PhD students, Ms. Daniela Schiavinato and Ms. Estefania Rojas Lopez, have joined our project as international students who both recently secured their student visas. They will embark on the</p>
</blockquote>
<blockquote>
<p>vital "Beyond Prediction" project, focusing on the signal-to-noise aspect in the upcoming months. Our team has welcomed DJ Champion, an assistant research fellow. DJ has previously been engaged in the successful completion of the MBIE smart idea project led by Plant &amp; Food Research, titled "Targeting Specialty: Taxonomically Restricted Genetics for Accelerated Selection". His contributions have been instrumental in finalizing that project, and he has also made significant discoveries regarding the accuracy issues with tools employed in lncRNA identification which we are pursuing further.</p>
<p>During the year we are reporting on, we have carried out an extensive simulation study to benchmark and validate a wide range of machine learning approaches with respect to their performance on GWAS data. We discovered that for existing AI models such as linear transformers the UKBB dataset is too small to outperform more standard machine learning approaches such as random forests. These results are now written up and being submitted to top machine learning conferences. A PhD student working on this project with me (Kieran Collienne) has been invited to present his results at Fred Hutchinson Cancer Research Center in Seattle.</p>
<p><strong>Research Outputs</strong></p>
<p><strong>Publications:</strong></p>
</blockquote>
<blockquote>
<p>Gardner PP, Paterson JM, McGimpsey S, Ashari-Ghomi F, Umu SU, Pawlik A, Gavryushkin A, Black MA. Sustained software development, not number of citations or journal choice, is indicative of accurate bioinformatic software. Genome biology. 2022 Dec;23(1):1-3. <u>https://doi.org/10.1186/s13059-022-02625-x</u> This article has received significant attention, boasting an impressive 'Attention Score' as a preprint, ranking it within the top 5% of all articles assessed by the AltMetric resource. Additionally, it has amassed 17 citations on GoogleScholar, encompassing references to both the article and its preprint version.</p>
</blockquote>
<blockquote>
<p>Te Aika B, Liggins L, Rye C, Perkins EO, Huh J, Brauning R, Godfery T, Black MA. Aotearoa genomic data repository: An āhuru mōwai for taonga species sequencing data. Molecular Ecology Resources. 2023 Sep 15. <u>https://doi.org/10.1111/1755-0998.1386</u></p>
</blockquote>
<blockquote>
<p>Bryant, D. and Huson, D. 2023. NeighborNet: improved algorithms and implementation. Frontiers in Bioinformatics 3. <u>https://www.frontiersin.org/articles/10.3389/fbinf.2023.1178600</u></p>
</blockquote>
<blockquote>
<p>Snyman, J., Fox, C. and Bryant D. 2023. Parsimony and the rank of a phylogenetic flattening. Journal of Mathematical Biology 86(44</p>
</blockquote>
<blockquote>
<p>Paper: Testing for phylogenetic signal in single-cell RNA-seq data. Jiří Moravec, Rob Lanfear, David Spector, Sarah Diermeier, and Alex Gavryushkin. Journal of Computational Biology 2023. <u>http://doi.org/10.1089/cmb.2022.0357</u></p>
</blockquote>
<blockquote>
<p>Paper: A fast lasso-based method for inferring higher-order interactions. Kieran Elmes, Astra Heywood, Zhiyi Huang, and Alex Gavryushkin. PLOS Computational Biology 2022. <u>https://doi.org/10.1371/journal.pcbi.1010730</u></p>
</blockquote>
<blockquote>
<p>Conference paper: SNVformer: an attention-based deep neural network for GWAS data. Kieran Elmes, Diana Benavides-Prado, Neset Ozkan Tan, Trung Bao Nguyen, Nicholas Sumpter, Megan Leask, Michael Witbrock, and Alex Gavryushkin. ICML Workshop on Computational Biology 2022. <u>https://icml-compbio.github.io/2022/papers/WCBICML2022_paper_58.pdf Queenstown Molecular Biology Conference</u> (QMB 2023) (international conference, this year included a Nobel medal holding participant). Invited talk: <u>Model- and data-driven approaches for genotype-phenotype data</u>.</p>
</blockquote>
<blockquote>
<p><u>The 28th annual RNA Society Meeting "RNA 2023" (Singapore)</u>. Poster: Single-cell RNA-seq phylogenomics.</p>
</blockquote>
<blockquote>
<p><u>The 26th Annual NZ Phylogenomics.</u> Talk: <u>Meeting Online algorithms and cancer evolutio</u></p>
</blockquote>
<blockquote>
<p><strong>Research talks:</strong></p>
</blockquote>
<blockquote>
<p>Parsimony and the rank of a phylogenetic flattening, DECO’23: Annual Phylogenetics Meeting, Bryant &amp; Gardner</p>
</blockquote>
<blockquote>
<p>The genetic robustness of RNA and protein from evolutionary, structural and functional perspectives, DECO’23: Annual Phylogenetics Meeting, Bryant &amp; Gardner</p>
</blockquote>
<blockquote>
<p>ICIAM’23. International Congress on Industrial and Applied Mathematics. Bryant presented an invited mini-symposium talk on Snyman et al. 2023. ICIAM is the largest and most important international conference in applied and industrial mathematics.</p>
</blockquote>
<blockquote>
<p><strong>Preprints</strong>:</p>
</blockquote>
<blockquote>
<p>Ashari Ghomi F, Langridge G, Cain AK, Boinett CJ, Abd El Ghany M, Pickard DJ, Kingsley RA, Thomson N, Parkhill J, Gardner PP, Barquist L. High-throughput transposon mutagenesis in the family Enterobacteriaceae reveals core essential genes and rapid turnover of essentiality. bioRxiv.</p>
</blockquote>
<blockquote>
<p>2022:2022-10. https://doi.org/10.1101/2022.10.20.512852 Pre-print: Automated convergence diagnostic for phylogenetic MCMC analyses. Lars Berling, Remco Bouckaert, and Alex Gavryushkin. <u>https://doi.org/10.1101/2023.08.10.552869</u></p>
</blockquote>
<blockquote>
<p>Pre-print: Estimating the mean in the space of ranked phylogenetic trees. Lars Berling, Lena Collienne, and Alex Gavryushkin. <u>https://doi.org/10.1101/2023.05.08.539790</u></p>
</blockquote>
<blockquote>
<p>Pre-print: Ranked subtree prune and regraft. Lena Collienne, Chris Whidden, and Alex Gavryushkin. <u>https://doi.org/10.1101/2023.05.16.541039</u></p>
</blockquote>
<blockquote>
<p>Pre-print: Online Bayesian analysis with BEAST 2. Remco Bouckaert, Lena Collienne, and Alex Gavryushkin. <u>https://doi.org/10.1101/2022.05.03.490538</u></p>
</blockquote>
<blockquote>
<p><strong>Example 3: Expanding our funding base and team</strong></p>
<p><strong>The Paul Allen AI Institute in Seattle has partnered with us on our work in verifying scientific claims, and is committing 0.5 FTE (split between two individuals) to this project, without any cost to us.</strong> See the section above on Transparent Data Science in Strategic Intent for more details of this work.</p>
<p><strong>We have successfully leveraged the progress in Beyond Prediction to acquire two more related grants, both of which help to expand our team and our impact.</strong></p>
</blockquote>
<ol type="1">
<li><p>AI models for <em>genome-wide association</em> studies (GWAS) data. Based on the Beyond Prediction progress and team connection, AIs Alex Gavryushkin &amp; Michael Witbrock) were successful in obtaining a Marsden Standard grant to add two PhD students to the team. During the year we have carried out an extensive simulation study to benchmark and validate a wide range of machine learning approaches with respect to their performance on GWAS data. We discovered that for existing AI models such as linear transformers the UKBB dataset is too small to outperform more standard machine learning approaches such as random forests. These results are now written up and being submitted to top machine learning conferences.</p></li>
<li><p>Bayesian phylogenetics inference methods and online algorithms for reconstructing cancer evolution. This project also resulted in a Marsden Grant (PI Drummond, AI Gavryushkin), that adds two more PhD students. Work this year involves developing such methods for RNA-sequencing data. The main outcome this year are: (a) scRNA-seq data has been shown to contain evolutionary signal (this result <u>in the highly ranked Journal of Computational Biology -- JCB</u>), and we were the first group in the world to show this. The result has been picked up internationally and now there are two more papers published confirming our result. (b) We extended this result to include spatial transcriptomics data, that is, adding location information about where the cell samples are located in the tumor. Addition of spatial data improved phylogenetic accuracy of our inference algorithms. This result is currently being written up by a postdoc working with me (funded from elsewhere) and will be submitted by the end of the year, when the postdoc finishes. (c) We have built an algorithm for automatic detection of phylogenetic MCMC convergence. This effectively positively solves the problem of whether phylogenetic MCMC is possible in the online framework. Two preprints are submitted to top journals in the area, pending a decision (<u>https://doi.org/10.1101/2023.05.08.539790</u>; <u>https://doi.org/10.1101/2023.08.10.552869</u>).</p></li>
</ol>
<blockquote>
<p>We are also exploring opportunities to collaborate with the emerging national RNA platform on the above 2 projects</p>
<p><strong>Impact</strong></p>
<p><strong>Our science impact is clearly evidenced again by a long list of publications in highly-ranked journals, only some of which are included in this report.</strong> This year, as last year, our team has produced over 40 research articles, the majority of which are in very highly ranked journals and conference.</p>
<p>Arguably more important is the fact that now we are seeing some of our software development efforts bearing fruit and being launched into the world. We are committed to developing reproducible, open source data, methods and applications, and we provide three specific examples below in detail, available to other researchers in Aotearoa and beyond via <em>github</em>.</p>
</blockquote>
<blockquote>
<p><strong>Example 1: Data Science algorithms for Phylogenetic Research</strong></p>
<p>Over the past 12 months, our collaborative team, including Dr. Jonathan Klawitter (postdoc), A/Prof Alex Gavryushkin, and his Ph.D. student Lars Berling, has initiated a ground-breaking research programme on tractable tree distributions. Our efforts centre on creating efficient computational methods that will vastly improve our ability to compactly represent (and more important do so in a way that is explanatory) the structure of posterior probability distributions on the space of trees. Such advancements promise not only rapid computations but also explainable interpretations and decompositions of uncertainty concerning evolutionary hypotheses, all by leveraging the compact attributes of Tractable Tree Distribution (TTD) data structures. This innovative research programme harnesses our international expertise in Bayesian phylogenetic inference algorithms and the statistical geometry of tree space. Ultimately, our objectives are strongly aligned with the development of tools essential for real-time phylodynamics, infectious disease genomic surveillance, and evolutionary approaches to cancer research and treatment. From our initial groundwork, we have already pinpointed prospects for at least six manuscripts, of which three are in advanced stages of preparation. Furthermore, our research expansion has paved the way for the inclusion of a new Ph.D. student, Sophie Yang. We are confident that her research, titled "Computations on Tractable Tree Distributions for Posterior Summary and approximate Bayesian phylogenetic inference", will contribute to revolutionising the manner in which we represent, estimate, and visualise evolutionary and phylodynamic analyses.</p>
<p><strong>Example 2: Data Science for Biodiversity</strong></p>
<p>The second impact example is measuring genetic diversity and inferring adaptability over time and space. By measuring how genetic diversity is structured in space, and how this influences the ability of species to adapt to changing selection pressures, including climate change, we can inform conservation management decisions that may mitigate the effects of low genetic diversity for threatened species, and can offer insight into the ongoing threat of invasive species establishing in novel areas. We have identified and summarised common diversity metrics from both the genetic/ genomic and ecological literature and have implemented these in R. We have begun to apply these methods to an exemplar genomic dataset of the taonga bird species, the threatened hihi (<em>Notiomystis cincta</em>). Funding and connections with Genomics Aotearoa* have enabled the generation of a high quality genome assembly and population sequencing data for 250 hihi from an intensively monitored population, which our Data Science methods are beginning to be applied to. Eirian Perkins is leading ongoing methods and workflow development that ensure 'live science', such that new data can be imported as it becomes available. She has also commenced the collation of a list of open access datasets that these new methods can be applied to. A substantial associated <strong>R package</strong> for implementing these models is now available on Github: <u>https://primer-e.github.io/senlm/</u></p>
</blockquote>
<blockquote>
<p>In the past year, our Biological Data Science: Modelling biodiversity sub-theme has made exciting progress on our aims to develop reproducible, open source methods to measure genetic diversity and infer adaptability over time and space. We had a new PhD student, Hui Zhen Tan, join the project in November 2022 and she has already completed analyses of the recombination landscape in hihi (stitchbird; Notiomystis cincta). Recombination is a fundamental evolutionary process that ‘shuffles the deck’ of existing genetic variation, creating new combinations of variants. This has important implications for the adaptive potential of species. With this data, Eirian Perkins has built a population simulation model of the hihi metapopulation. This simulation creates an important test case for how we measure and predict adaptive potential in populations with more or less connectivity to other populations and varying population size over time. The simulations are ground-truthed from a large whole genome resequencing dataset of over 300 birds, plus a hihi genome assembly, funded by Genomics Aotearoa*. <strong>The exciting combination of methods from the Data Science project applied to data generated via Genomics Aotearoa demonstrates the active connection between and complementarity of the two projects.</strong></p>
<p>The <em>hihi</em> simulation allows us to test the impact of future management decisions on adaptive potential for this taonga threatened species. Moreover, it also provides an important template for application to any wild species where there is some information about population size and connectivity between them, supporting our overall project aim to measuring how genetic diversity is structured in space, and how this influences the ability of species to adapt to changing selection pressures, including climate change. Both Eirian and Hui Zhen are focused on producing reproducible, reusable code; for Eirian in particular this has</p>
</blockquote>
<blockquote>
<p>meant developing code that is containerised, and has ensured her workflows are set up such that new data can be imported as it becomes available, i.e. ‘live’.</p>
<p>In terms fo applications, this work provides a novel new tool that enables ecologists to track shifts in species’ latitudinal optima – a tell-tale of climate change at large spatial scales. We’ve created R packages for implementing the non-linear model framework, with inclusion of zero-inflated error distributions (for counts, percentage cover, biomass, traits, etc.) to handle a broad range of large-scale ecological data types. One R package fits flexible non-linear species-environment models (<u>senlm</u>) using a maximum likelihood approach; another R package (more recently) embraces a fully Bayesian approach, which leads the way for hierarchical (multi-layer) modelling of key parameters over multiple species, locations, time-frames, etc.</p>
<p>With postdoc Paulo Martins, we’ve used this modelling framework to analyse the effects of climate change on spatial distributions of hundreds of species of birds in North America. We found that one-third of the 209 species analysed showed significant shifts in optimum latitude over the past 50 years (predominant poleward shifts, particularly in Western regions), and we were able also to use these models to show that the sizes of shifts were linked to key physiological traits, most notably birds’ thermal optima – a clear signal of the effects of climate change.</p>
<p>This research has produced 8 papers and 12 seminars or talks over the reporting period. Importantly, it has also led to the the 'publishing' of related datasets and two open source R packages:</p>
<p>Rabel, H., Smith, A.N.H. and Anderson M.J. (2023). modskurt: Bayesian Distributions of Species Abundance Along Environmental Gradients. R package version 0.2.2. <u>https://hdrab127.github.io/modskurt/</u>.</p>
</blockquote>
<blockquote>
<p>Walsh, D., Anderson, M.J., Punnett A.J. and Adegoke, N. (2022). senlm: Species environment non-linear modelling. <u>https://primer-e.github.io/senlm/</u>.</p>
</blockquote>
<blockquote>
<p><strong>Example 3: LinguaPhylo (Lphy)</strong></p>
</blockquote>
<blockquote>
<p>LinguaPhylo (LPhy for short - pronounced el-fee) is a probabilistic model specification language to</p>
<p>concisely and precisely define phylogenetic models. The aim is to provide a language (work towards</p>
<p>a <em>lingua franca</em>) for probabilistic models of phylogenetic evolution that is independent of the method to</p>
<p>perform inference. This language is readable by both humans and computers. We reported on an earlier</p>
<p>version of Lphy last year, but now it is gaining traction as a vital new approach for studying phylogenetics.</p>
<p><u>https://linguaphylo.github.io/</u></p>
</blockquote>
<blockquote>
<p><u>LinguaPhylo: A probabilistic model specification language for reproducible phylogenetic analyses</u></p>
</blockquote>
<blockquote>
<p>Drummond AJ, Chen K, Mendes FK, Xie D (2023) LinguaPhylo: A probabilistic model specification language for reproducible phylogenetic analyses. PLOS Computational Biology 19(7):</p>
<p>e1011226. <u>https://doi.org/10.1371/journal.pcbi.1011226</u></p>
<p><strong>Investing in people (capability)</strong></p>
<p>Via our PhD and post-doc funding, we are growing a new generation of Data Science researchers with the skills necessary to make strong contributions in industry, academia and government, along with a passion for open and transparent science. Despite the handicaps associated with the pandemic, we are now almost up to strength with our post-doc appointments and are catching up to where we hoped to be with our PhD students.</p>
<p><strong>The 2022 Research Bazaar</strong></p>
<p>The 2022 <em>Research Bazaar</em> (<em>ResBaz’22</em>) event led by the University of Auckland (Laura Armstrong, Mark Gahegan and the team at the Centre for eResearch) along with contributions from The Universities of Otago and Wellington. As in 2021, we hosted a special track in Data Science sponsored by <em>Beyond Prediction</em> as part of our national outreach to increase data science literacy among the research community at large. This intensive 3 day event was delivered mostly online to allow us to reach a bigger</p>
</blockquote>
<blockquote>
<p>and wider audience but included a range of practical activities supported by gangs of tutors. The specific Data Science track offered 25 unique Data Science-focussed workshops , along with a broad mix of other digital skills offerings. Topics included:</p>
<p>Introduction to Machine Learning</p>
</blockquote>
<blockquote>
<p>Becoming a Data Scientist</p>
</blockquote>
<blockquote>
<p>Machine Learning in Julia</p>
</blockquote>
<blockquote>
<p>Python for image manipulation and repeatable research pipelines</p>
</blockquote>
<blockquote>
<p>Māori and Pacific researchers: Reimagining relationality with digital tools</p>
</blockquote>
<blockquote>
<p>Working with Social Media Data</p>
</blockquote>
<blockquote>
<p>Introduction to R and R-Studio</p>
</blockquote>
<blockquote>
<p>Open Source in Research</p>
</blockquote>
<blockquote>
<p>…and many others.</p>
</blockquote>
<blockquote>
<p>Half of the activities were effectively ‘tasters’ introducing new approaches, tools and data science skills, and half were ‘hands-on’ with varying levels of complexity from beginner to advanced, so researchers got to learn new skills that they can apply directly in their day-to-day research.</p>
<p><strong>This year we set a new record with over 5,000 registrations from over 1,400 unique individuals. Looking at affiliations, every single NZ university and CRI was represented by the attendees. The following organisations also were represented. We are really pleased that this skills training now stretches well beyond the universities and CRIs.</strong></p>
</blockquote>
<table>
<tbody>
<tr>
</tr>
<tr>
</tr>
<tr>
</tr>
<tr>
</tr>
<tr>
</tr>
<tr>
</tr>
<tr>
</tr>
<tr>
</tr>
<tr>
</tr>
</tbody>
</table>
<blockquote>
<p><strong>Adding PhD students and research fellows to our team</strong></p>
<p>Over the past 12 months, we have seen the flow of PhD students into Aotearoa finally pick up, so we have been recruiting. We have added two PhD students and a research fellow (<strong>D.J. Champion</strong>) to the team at Otago, and two new PhD students along with a new post-doc (<strong>Jonathan Klawitter</strong>) Auckland. Two further post-docs have been identified at Auckland--both of whom are currently finishing up their PhDs with Beyond Prediction.</p>
</blockquote>
<blockquote>
<p><strong>Domestic and international collaboration</strong></p>
<p><strong>Domestic Collaboration</strong></p>
<p>There is now a strong collaboration on AI in genomics between the groups led by professors Alex G (U.Canterbury) and Michael Witbrock (U.Auckland), both PIs in this project but with no previous history of working together. This has led to a successful Marden fund application now underway to develop novel computational tools for learning the genetic variation.</p>
</blockquote>
<blockquote>
<p>Close collaboration on the next release of this product with <em><strong>Stencila</strong></em>, a New Zealand company focused on interactive and collaborative publishing (https://stencila.io/). We are now embedding some of Stencila's online, collaborative publishinng technology into our Live Data Science project. We have deepened our relationship with ESR and continue to host an ESR employee (Hugo Strydom) as a PhD student with us, cementing lines of collaboration over infectious disease monitoring and modelling. Hugo is a senior scientists at ESR, registered at Massey University and co-supervised by a team from Massey, ESR and the University of Auckland. The initial phase of his work is focussed on the phylodynamics of <em>Salmonella enterica</em>Enteritidis sequence type ST11, which has been the cause of an ongoing outbreak of salmonellosis in Aotearoa associated with eggs and poultry meat since 2019. He has been working on developing his skills in bioinformatics, ancestral state reconstruction modelling and data visualisation using a whole genome sequenced dataset of ~250 isolates from human cases and poultry samples.</p>
</blockquote>
<blockquote>
<p><strong>International Collaboration</strong></p>
<p>The Paul Allen Institute for Artificial Intelligence (Allen AI) is partnering with us on our project evaluating the trustworthyness of scientific claims, using deep learning methods. Allen AI are contributing about 0.5 FTE along with a great deal of experience and standing into this project at no cost to us. Allen AI is perhaps the leading AI research institute anywhere in the world.</p>
</blockquote>
<blockquote>
<p>We have begun a collaboration with Prof. Ian Foster at the Argonne National Labs in Chicago, developing better tools for capturing and packaging the complex scientific workflows that are often associated with data science research. These workflows now form the basis of our 'Live Publication' research--one of our key projects in the grant. <em><strong>We are working together weekly on integration tasks with Argonne's Globus team.</strong></em></p>
</blockquote>
<blockquote>
<p>International interest from: Carole Goble's eScience group at U. Manchester, around the LivePublication project has led to a visit to Manchester (Mark Gahegan) and introductions to several other research in the EU whoare keen on working with us on this stretch goal.</p>
</blockquote>
<blockquote>
<p>Coordination with international consortium on open data (and research artefact) publication, RO-Crate (https://www.researchobject.org/ro-crate/)</p>
</blockquote>
<blockquote>
<p>We continue to collaborate with the Alan Turing Institute in the UK on the Machine Learning in Julia (MLJ) project: https://towardsdatascience.com/machine-learning-in-julia-5bca700e0348. The Julia languiage has the ease of use of scripting in Python, but (unlike Python) also offers very high levels of computational performance and efficiency. An increasing international community is using these tools now. The Technische Universität Kaiserslautern-LandauUniversity in Germany is now supporting the development of MLJ via a significant funding agreement that pays for our senior research fellow,Anthony Blaom.</p>
</blockquote>
<blockquote>
<p>Alexei Drummond visited the Posada Lab at University of Vigo, Spain, to carry out joint work on the evolution of genomes.</p>
</blockquote>
<blockquote>
<p><strong>Vision Mātauranga</strong></p>
<p>Research fellow <strong>Alistair Lamont</strong> has been working closely with Profs <strong>Phillip Wilcox</strong> and <strong>Mik Black</strong> to develop a new, efficient, genome simulator that can incorporate key geographic and structural parameters. This is the next step in the design of tools appropriate for genomic analysis and prediction within specific culture communities, particularly iwi not well-served by existing tools with generic assumptions and potential euro-centric data biases. The simulator will also be critical for the design and implementation (in collaboration with Bryant and Gardner) of new analytical tools.</p>
<p><strong>Phil Wilcox</strong> has also given several talks across NZ and wider on the important considerations of Precision Medicine, from a Māori Data Sovereignty perspective. His population simulation project as part of this grant complements the research he is doing on the <em>Rakeiora</em> and <em>Variome</em> projects that are funded via Genomics Aoteatoa. <strong>Wilcox</strong> also gave an invited talk on this work at the high prestigious EMBL</p>
</blockquote>
<blockquote>
<p>conference: entitled ‘When less isn’t more: the value of indigenous knowledge in understanding the relationship between the organism and its environment’. The figure below shows the key messages from that talk and how his with for us on population simulation studies helps to optimise study design.</p>
</blockquote>
<blockquote>
<p>Working in consultation with the Māori Data Sovereignty Network (Prof. Maui Hudson) we have added significant new materials on Māori Data Sovereignty into our Research Data Management Workshop, that was offered widely to researchers across the country earlier this year as part of our Data Science-themed Research Bazaar (ResBaz).</p>
<p>We have been working on preparing our Māori summer scholarships programme, which will run throughout the summer of 2023. Advertisements for expressions of interest will from undergraduates (at any stage in their studies) will be posted shortly. All students admitted will receive a basic scholarship, be attached to one of Beyond Prediction's core projects and will have a Māori mentor as well as other supervision.</p>
<p><strong>Additional information</strong></p>
</blockquote>
