<h2 id="public-statement">Public Statement</h2>
<p>Our overall aim is to develop new data science methods that help to <strong>improve the process of conducting and reporting research</strong>, and to use these developments to <strong>improve our national capability in genomics and population health and ecology research</strong>, supporting our bio-economy and at the same time improving health and environmental outcomes.</p>
<p>The team has achieved significant research progress this year in our three distinct areas:</p>
<ol type="i">
<li><p><strong>bioinformatics</strong>—studying the relationships between genomics and disease to help improve our response to agricultural disease outbreaks and human health outcomes;</p></li>
<li><p><strong>ecological modelling</strong>—creating new models to study the entire genome of threatened and invasive species to assess ecological impact and remediation;</p></li>
<li><p><strong>live and transparent data science</strong>—creating new ways of publishing and updating science studies to keep them up to date as new data and methods emerge, reducing the time that researchers spend updating, re-running and reporting their findings.  In each of these areas we have developed new methods and related software that has been shared openly with the science community.</p></li>
</ol>
<p>Collaborations within our team are now leading to additional grant proposals in the areas of AI and genomics, and we have strengthened our collaborations with other research partners based in Aotearoa: notably <strong>Public Health Science</strong> (formerly EHR), <strong>Genomics Aotearoa</strong>, the <strong>RNA Platform</strong>, <strong>DoC</strong>, <strong>MPI</strong>, <strong>MoH</strong> and overseas: <strong>The Allen AI institute</strong> and <strong>Argonne National Labs</strong> - USA, <strong>Federal Institute for Risk Assessment (BfR)</strong> and the <strong>Robert Koch Institute</strong> - Germany, <strong>Institute of Zoology</strong> - UK, and <strong>ETH</strong> -Switzerland).</p>
<p>This year our outreach &amp; training event—<strong>ResBaz 2025</strong> —provided hands-on data science training to over 2300 individual researchers throughout the country, that they can apply in their own research.  We are also training a future generation of NZ data scientists via our summer placements, PhD scholarships and post-doctoral appointments.  </p>
<h2 id="strategic-intent">Strategic Intent</h2>
<p>We have placed more emphasis on AI methods in science over the last year. These featured previously, but now of course the number of opportunities in. this area is increasing rapidly. We are now using generative AI in our work on Transparent &amp; Trusted research, in our project that dynamically updates research reports and in our bioinformatics research.</p>
<p>Our strategic direction focuses on developing new data science and AI methods that improve the conduct of research, and to showcase some of these improvements in action, in our specialist areas of: Biology, Genomics, Population Health and Ecology.  We provide a high-level summary of our work towards these strategic directions below.</p>
<p>Our research on trustworthy, explainable artificial intelligence aligns directly with MBIE’s goal of enabling a digital economy built on trust and transparency. We have begun a new project focused on hallucination detection and mechanistic interpretability, research that helps us move past treating artificial intelligence models as “black boxes” to uncover how and why they arrive at their outputs. By identifying the internal circuits and representations that drive their behaviour, we can better improve and audit them in a scientifically rigorous way. For New Zealand, developing innovation and expertise in this field is particularly important: it ensures that we are not just adopters of overseas AI technology, but leaders in shaping it and applying it responsibly.</p>
<p>Our <strong>Live Data Science </strong>initiative has now dramatically <strong>closed the gap between the ‘doing’ of science and the ‘publishing’ of science.  </strong>We have now developed workflows that update ‘live’ research documents in real time, in response to external triggers such as new data, better algorithms, or further experimentation. In an era when data are ubiquitous and regularly updated, there need no longer be a lag between writing and publishing science—whether in academic journals or in reports for business or government.  Furthermore, since this process is now automated and supported by generative AI, the human effort required to update reports has been significantly reduced, saving time for researchers and enabling them to be more productive. This work supports New Zealand’s broader strategy in AI for education and research and contributes to international discussions on trustworthy AI and responsible innovation.</p>
<p>Our <strong>Transparent Data Science</strong> improves the trustworthiness and reproducibility of data-intensive research, providing tools to question scientific claims and query research processes.  This opens up research to more scrutiny, which helps to improve confidence, that claims made about research hold water and are not exaggerated. and offers new avenues for public engagement. The tool we have built here is currently being evaluated for commercialisation and a limited rollout for evaluation. It is unique, in that it requires no training, and it reports not only its inner reasoning but also its confidence in its findings.</p>
<p>Our work applying data science methods in the <strong>Life Sciences</strong> tackles a multitude of longstanding analytical challenges, including: (i) new statistical techniques to improve the accuracy of phylodynamic modelling, with applications into population health and infectious diseases, (ii) modelling the entire genome of a species, not just an individual, and using this genome to anticipate adaptation capacity to a changing environment (iii) understanding the complex relationships between population genomics and common illnesses, to better target health interventions and (iv) better understanding and thus ability to control the spread of infectious diseases through susceptible populations. Some particular highlights for the past year include:</p>
<ul>
<li><p>Developing Bayesian Model Averaging (BMA) methods for parametric coalescent models in population genetics, advancing statistical techniques for dealing with model uncertainty in biological inference. This work contributes to more reliable understanding of population dynamics and demographic history reconstruction using genetic data.</p></li>
<li><p>Working on joint inference of genotype variants and cell lineage phylogeny and developing the calibrated coalescent point process - both advancing computational methods for complex biological data analysis.</p></li>
<li><p>Developing regularised conditional clade distributions to enhance phylogenetic posterior estimation, improving the accuracy and reliability of evolutionary inference.</p></li>
<li><p>Building phylogenetic inference methods through work on detecting rogue taxa and credible set estimation.</p></li>
</ul>
<p>We have also started a new workplan that buildis individual-based genetic models that can assess and predict the ability of invasive species to adapt and expand their range, which has significant implications for understanding the likelihood of new biosecurity incursions, the further spread of already resident invasive species, and their subsequent economic and ecological impact.</p>
<p>Progress such as this not only moves forward our understanding of genomics and phylogenetics, but also provides better tools to our practicing bioinformatics researchers, working to support New Zealand’s bio-economy.</p>
<h2 id="impact">Impact</h2>
<p>Our modelling biodiversity sub-theme has continued to develop reproducible, open source methods to measure genetic diversity and infer adaptability over time and space. Our research to understand the adaptive capacity of threatened species, using the <em>hihi</em> - stitchbird as a model system, has informed the development of management strategies for <em>hihi</em> and has also developed expertise in our team that is being utilised for species recovery more generally. For example Anna Santure was invited by the Department of Conservation to provide expert advice to genetic management strategies for our rarest bird, t<em>ara iti</em> (NZ fairy tern), and was also invited onto a Genomics Aotearoa project to understand the genetic underpinnings of disease risk in <em>hoiho</em> (yellow eyed penguin), which have faced extreme population losses on mainland New Zealand while maintaining relatively healthy populations in the Subantarctic islands. This work makes a key contribution to understanding how DNA sequencing costs can be minimised while obtaining sufficient information to be able to conduct genomic analyses by 'imputing', (i.e., inferring) missing data.</p>
<p>Over the last two years we have been applying some of our new computational scaling methods to Bayesian phylogenetics. We have developed a new method for accurately estimating the distribution of phylogenetic trees of a phylogenetic analysis. This new method provides better probability estimates and more accurate summary trees compared to previous state-of-the-art techniques.  We have published these results and the methods are now implemented within the BEAST2 software (<a href="https://www.beast2.org/">https://www.beast2.org/</a>), now made available for other researchers and industry to use. Our work significantly improves a longstanding method, and we expect the new summary tree method to positively impact hundreds of phylogenetic research projects per year.</p>
<p>The following GitHub repositories have been developed and deployed:</p>
<ul>
<li><p><a href="https://github.com/EvaLiyt/VariantCallingExperiments.git">https://github.com/EvaLiyt/VariantCallingExperiments.git</a></p></li>
<li><p><a href="https://github.com/moverwater/calibrated_cpp.git">https://github.com/moverwater/calibrated_cpp.git</a></p></li>
<li><p><a href="https://github.com/moverwater/CalibratedCPP_BEAST.git">https://github.com/moverwater/CalibratedCPP_BEAST.git</a></p></li>
</ul>
<p>Within the last year our <em>LivePublication</em> project, a standards-based framework which connects experiment infrastructure with procedural dynamic research articles, has matured with the development of open-source software and a fully realised case study aimed at detecting changes in New Zealand’s tides, beaches and coastal reaches. This environmental exemplar aligns with national priorities in climate resilience and sustainable land use and has provided the first public deployment of LivePublication. With the framework now stable and domain-agnostic, we are ready to onboard additional high-impact case studies with partner groups.  All code used to develop and create LivePublication is open and freely reusable and reproducible.</p>
<p>The <em>CoastSat</em> LivePublication can be found here <a href="https://coastsat.livepublication.org/"><u>https://coastsat.livepublication.org/</u></a>. The expertise gained on making the <em>CoastSat</em> LivePublication using the framework will allow us to help other researchers and projects to make use of this to more efficiently and regularly update time-critical reports and research investigations. </p>
<p>The CoastSat implementation is available OpenAccess on Github, and hosted here:</p>
<ul>
<li><p><a href="https://coastsat.livepublication.org/"><u>https://coastsat.livepublication.org</u></a></p></li>
<li><p><a href="https://github.com/GusEllerm/CoastSat-interface.crate"><u>https://github.com/GusEllerm/CoastSat-interface.crate</u></a></p></li>
<li><p><a href="https://github.com/GusEllerm/CoastSat-micropublication"><u>https://github.com/GusEllerm/CoastSat-micropublication</u></a></p></li>
<li><p><a href="https://github.com/GusEllerm/CoastSat-shorelinepublication"><u>https://github.com/GusEllerm/CoastSat-shorelinepublication</u></a></p></li>
<li><p><a href="https://github.com/GusEllerm/CoastSat/tree/livepublication"><u>https://github.com/GusEllerm/CoastSat/tree/livepublication</u></a></p></li>
</ul>
<p>We have also implemented a provenance tracking system that annotates AI-generated versus human-written content. In addition, we developed a prototype GraphRAG database and DocSQL query engine with Kùzu DB, enabling AI to query research literature more accurately and transparently. These tools were shared in workshops and demos with positive researcher feedback, and our open-source releases on GitHub have begun attracting community engagement.</p>
<p>Kernel-Enriched AI (KEA) Explain is a neurosymbolic technique we have developed for detecting and explaining hallucinations in large language models. It provides not only improved accuracy in identifying errors but also explanations that help users understand why a response may be unreliable. The application of this method can lead to more trustworthy adoption of AI in sensitive domains such as government, education, and health, where factual reliability is critical.</p>
<p>We are working to understanding how knowledge is reorganised inside neural networks during the process of distillation, where smaller AI models are trained from larger ones. The impact of this work lies in improving the design and evaluation of smaller, more efficient AI systems. For New Zealand, this has particular significance: <u>distilled models reduce computational and energy demands</u>, making advanced AI systems viable for organisations with limited resources, including small businesses, government agencies, and researchers. Yet, little is known about how the internals of the models change structurally during the distillation process; our work is the first to do that. Developing expertise in this area positions New Zealand to build and deploy reliable AI that is not only powerful but also sustainable and cost-effective.</p>
<p>This work has been published (see excellence below) and shared as open source software: <a href="https://github.com/Reih02/hallucination_explanation_graph_kernel_analysis">https://github.com/Reih02/hallucination_explanation_graph_kernel_analysis</a>.</p>
<p>Our scientific claim extraction work has already been tested on large scientific corpora and provides a scalable pipeline for knowledge organization. The research supports New Zealand’s reputation in advancing AI for science, contributing to international efforts on research integrity and scientific discovery. Our open-source datasets and code for claim extraction/verification workflows are publicly shared via GitHub to encourage adoption.</p>
<h2 id="excellence-science-excellence">Excellence (Science excellence)</h2>
<p><strong>Excellence in published research outcomes</strong>  </p>
<p>The team has produced over <strong>30 peer-reviewed research papers and conference presentations</strong> in this reporting year. Many of these are published in the very top journals and conferences.  Here is just a brief selection of some of the highlights.</p>
<p><strong>Research papers published &amp; presented</strong></p>
<ul>
<li><p>Hui Zhen Tan (2025). Inbreeding load in a small and managed population: two decades of Hihi/Stitchbird genomics. <em>Genetics Society of Australasia Conference</em>, Auckland, [<strong>Best talk prize</strong>].</p></li>
<li><p>William Pearman (2025). Joint consideration of selection and microbial generation count provides unique insights into evolutionary and ecological dynamics of holobionts Genetics Society of AustralAsia conference, Auckland, July 2025: William Pearman,</p></li>
<li><p>Tan, H.Z., Stuart, K.C., Vi, T.; Whibley, A., Bailey, S., Brekke, P., <u>Santure, A.W.</u> (2025). International journal of <em>Molecular Ecology Resources</em>.</p></li>
<li><p>Z Li, H Ning, S Gao, K Janowicz, W Li, ST Arundel, C Yang, B Bhaduri, M.Gahegan (2025). Giscience in the era of artificial intelligence: A research agenda towards autonomous GIS. Annals of GIS. arXiv preprint arXiv:2503.23633.</p></li>
<li><p>K Sila-Nowicka, A Moore, D O'Sullivan, B Adams, M Gahegan (2025). LIPIcs, Volume 346, GIScience 2025, proceedings 13th International Conference on Geographic Information Science.</p></li>
<li><p>NÖ Tan, N Tandon, D Wadden, O Tafjord, M Gahegan, M Witbrock (2024). Faithful reasoning over scientific claims, Proceedings of the AAAI Symposium Series 3 (1), 263-272.</p></li>
<li><p>Y Chen, X Lin, B Yan, L Zhang, J Liu, NÖ Tan, M Witbrock (2024). Meta-inverse reinforcement learning for mean field games via probabilistic context variables. Proceedings of the <em>AAAI Conference on Artificial Intelligence</em> 38 (10).</p></li>
<li><p>G. Ellerm, M. Gahegan, B.Adams (2025) From Static to Dynamic: Live Publication and the Quest for Reproducible, Living Articles. <em>NZ eResearch NZ Conference</em>.</p></li>
<li><p>G. Ellerm, M. Gahegan, B.Adams (2025) Reproducibility in GIscience: how we can leverage reproducibility to enliven research articles. <em>13th International Conference on Geographic Information Science</em>.</p></li>
<li><p>R. Haskins, B. Adams. (2025) KEA Explain: Explanations of Hallucinations using Graph Kernel Analysis, in <em>Proceedings of the 19th Conference on Neurosymbolic Learning and Reasoning (NeSy 2025)</em>, Santa Cruz, CA.</p></li>
<li><p>For the above we have been invited to submit an extended version to the <em>Neurosymbolic Artificial Intelligence Journal</em> with planned submission by end of August 2025.</p></li>
<li><p>R. Haskins, B. Adams (under review) Distilled Circuits: A Mechanistic Study of Internal Restructuring in Knowledge Distillation, submitted to <em>The Thirty-Ninth Annual Conference on Neural Information Processing Systems (NeurIPS 2025)</em>, San Diego, CA.</p></li>
<li><p>Gardner PP. <a href="https://doi.org/10.1093/bioadv/vbaf011">A bioinformatician, computer scientist, and geneticist lead bioinformatic tool development—which one is better?</a> Bioinformatics advances. 2025;5(1):vbaf011.</p></li>
<li><p>Champion DJ, Chen TH, Thomson S, Black MA, Gardner PP. <a href="http://dx.doi.org/10.1261/rna.080416.125">Evaluating computational tools for protein-coding sequence detection: Are they up to the task?</a> RNA. 2025 Jun 11:rna-080416.</p></li>
<li><p>Contracts with the RNA Platform for the “Target Selection” and “RNA Optimisation” Pillars.</p></li>
<li><p>Gardner PP (2025), <a href="https://www.youtube.com/watch?v=Z-eIFn5muJU">What drives bioinformatic tool accuracy?</a> NZ eResearch Conference.</p></li>
<li><p>Klawitter et al, 2024): "Skeletons in the Forest: Using Entropy-Based Rogue Detection on Bayesian Phylogenetic Tree Distributions".- <a href="https://www.biorxiv.org/content/10.1101/2024.09.25.615070v1.article-info">https://www.biorxiv.org/content/10.1101/2024.09.25.615070v1.article-info</a></p></li>
<li><p>Klawitter et al, (2024: "Credible Sets of Phylogenetic Tree Topology Distributions" - <a href="https://arxiv.org/abs/2505.14532v1">https://arxiv.org/abs/2505.14532v1</a>.</p></li>
<li><p>High imputation accuracy can be achieved using a small reference panel in a natural population with low genetic diversity. <em>Molecular Ecology Resources </em>e70024. <u><a href="https://doi.org/10.1111/1755-0998.70024">https://doi.org/10.1111/1755-0998.70024</a>.</u></p></li>
</ul>
<ul>
<li><p>Genetics Society of Australasia conference, Auckland, July 2025 – Development of a Population Simulator to Optimise Study Designs and Estimate Polygenic Disease Risk/Resilience in Aotearoa Māori Populations</p></li>
<li><p>Maurice Wilkins Centre Symposium, Auckland, February 2025 – Development of a Population Simulator to Optimise Study Designs and Estimate Polygenic Disease Risk/Resilience in Aotearoa Māori Populations</p></li>
<li><p>eResearch conference, Christchurch, February 2025 - Population Simulation to Optimise Study Designs and Estimate Polygenic Disease Risk/Resilience in Aotearoa Māori Populations</p></li>
<li><p>GeneMappers, Christchurch, August 2024 - Population Simulation to Optimise Study Designs and Estimate Polygenic Disease Risk/Resilience in Aotearoa Māori Populations</p></li>
</ul>
<ul>
<li><p>Yuan Xu presented poster at SMBE annual meeting in Beijing (July 2025), attracting significant interest from international researchers.</p></li>
<li><p>Sophie Yang presented at SMBE conference, highlighting impact to global audience of experts.</p></li>
<li><p>Eva Li presented a flash talk at SMBE conference on her simulation pipeline for single-cell DNA sequencing variant calling.</p></li>
</ul>
<p><strong>Invitations to speak, keynote presentations, media presentations</strong></p>
<ul>
<li><p>Anna Santure was invited to give a seminar to the Ecology and Evolutionary Biology Cluster, School of Biosciences, University of Sheffield, July 2024</p></li>
<li><p>Invited speaker at the Association of Biosafety for Australia &amp; New Zealand Conference, Sydney, November 2025 to discuss potential harmful applications of AI in biotechnology.</p></li>
</ul>
<ul>
<li><p>Mark Gahegan was invited to speak on “AI for evaluating scientific claims” at the <em>Trillion Parameters Consortium</em> Special Interest Group at <em>Supercomputing Asia’25.</em></p></li>
<li><p>Mark Gahegan gave an invited seminar at Penn State University in the USA on “AI and the future of research”</p></li>
<li><p>Co-PI Lara Greaves is editing a special issue of the Journal of the Royal Society of NZ on Māori Data Sovereignty, with 30 expressions of interest for 12 spots. Attended and presented at the Global Indigenous Data Sovereignty conference in Canberra in April</p></li>
<li><p>Lara also organised a strategy hui for <em>Te Mana Raraunga</em> in Feb (Waikato Uni, 100 members, in person) and is leading a panel for <em>International Data Week</em> in Brisbane set for October, on <em>Te Mana Raraunga</em> and the CARE principles </p></li>
</ul>
<h2 id="research-collaborations-eu-horizons-and-co-fundingleverage"><strong>Research Collaborations</strong> (EU Horizons and co-funding/leverage)</h2>
<p>Co-PI Anna Santure was awarded a two year Faculty Research Development Fund Postdoctoral Award from the Faculty of Science, University of Auckland. This grant has employed research fellow Tram Vi who has built a pipeline to simulate genetic data from populations and apply different algorithms to find the best approach to impute, or infer, missing genetic data when cost-effective low coverage genetic sequencing is performed. This funding builds on research lead by our Biological Data Science: Modelling biodiversity PhD student Hui Zhen Tan that has resulted in our recently accepted publication (<u><a href="https://doi.org/10.1111/1755-0998.70024">https://doi.org/10.1111/1755-0998.70024</a>)</u>. </p>
<p>We are collaborating with international researchers working on fact-checking, misinformation detection, and AI for scientific discovery, and exploring potential synergies with EU-funded projects on trustworthy AI and digital science infrastructures.</p>
<h2 id="capability-development-investing-in-people">Capability Development (Investing in people)</h2>
<p><strong>ResBaz 2025</strong></p>
<p>Once again, we sponsored and helped to deliver a significant national outreach &amp; training event: ResBaz 2025.  This event was attended by over <strong>2300 unique researchers, and was our biggest audience yet.</strong> Attendees all learned <strong>new data science skills</strong> that they can apply to their own research.  We offered <strong>47 workshops</strong> this year (some more than once), over 5 days, each session delivered by expert instructors and trainers. Many workshops introduced hands-on, practical data science exercises to upskill our national science community at large.</p>
<p>Attendees were from <strong>all of Aotearoa's universities and most of our CRIs, some from government and some from the private sector</strong>.  A full programme for the event can be found here: <a href="https://resbaz.auckland.ac.nz/schedule/">https://resbaz.auckland.ac.nz/schedule/</a>.  Feedback from the attendees was outstanding, this is a very useful and impactful event that has a marked impact on our national data science capability.</p>
<p><strong>PhD Student Highlights</strong></p>
<p>Below are some of the highlights from the progress of some of the PhD and post-doctoral scholars funded in this project, as they progress through their studies with the guidance of our investigators. These people are acquiring new skills in data science in application areas important to New Zealand’s industries and environment. We have confidence that as they complete their training, they take their place in the next generation of our science workforce.</p>
<ul>
<li><p>We are delighted to report that our research fellow Dr William Pearman secured a highly competitive four year Mana Tūāpapa Future Leader Fellowship. Dr Pearman continues as a 0.2 FTE Biological Data Science research fellow while holding the 0.8 FTE Mana Tūāpapa fellowship. This arrangement has supported William to be able to be full time and to continue to work on his data science <em>holobiome</em> project</p></li>
<li><p>PhD student Hui Zhen Tan has just had the second manuscript from her PhD published online, and won a conference talk award for presenting this work at the recent Genetics Society of Australasia conference.</p></li>
<li><p>We have also recently employed an exceptional research fellow, Dr. Kamolphat Atsawawaranunt. As part of this new fellowship position, Kamolphat has already submitted a manuscript, currently under review, and has designed and is now running individual-based genetic models that can assess and predict the ability of invasive species to adapt and expand their range.</p></li>
<li><p>Our former PhD student, Eirian Perkins, has just taken up an incredible career opportunity to work at Rocket Lab.</p></li>
<li><p>The PhD students, Daniella Schiavinato and Estefania Rojas Lopez are both making excellent progress in the second year of their project. Both have developed good coding skills and are learning robust machine learning approaches for their genomics projects.</p></li>
<li><p>DJ Champion (Assistant Research Fellow) published his first paper on the accuracy of protein annotation tools, a key finding is that many “modern” methods (SVMs, neural networks) due to mismatches between training and application data, poor training and test data selections and numerous other poor analytic decisions. We have now pivoted his work to a project that aligns with the RNA Platform, investigating the contentious area of whether RNA secondary structure has an impact on messenger RNA degradation rates. This work fits neatly with the “signal-to-noise” theme, and has direct applications for the RNA Platform.</p></li>
<li><p>Gus Ellerm, a PhD candidate associated with this grant, is in the final stages of completing his dissertation. As part of his dissertation, Gus created the LivePublication Framework and implemented the <em>CoastSat</em> application described above.</p></li>
<li><p>Sarah Hannah is in the second year of her project on the phylodynamics of <em>Neisseria meningitidis, </em>the cause of meningococcal disease. She has already completed an analysis of the population structure and genomic epidemiology of one of the most important clades circulating in Aotearoa, and the effect of Covid-19 lockdowns on the transmission dynamics of this clade. This work is being prepared for publication.</p></li>
<li><p>Hugo Strydom is in his final year and plans to submit before the end of this year. His programme of work is on the phylodynamics of enteric pathogens <em>Salmonella enterica</em> Enteritidis, and <em>Yersnia enterocolitica</em>. Both pathogens have caused major outbreaks of disease in Aotearoa. He awaiting the final decision on his first manuscript “Temporal reconstruction of a <em>Salmonella</em> Enteritidis ST11 outbreak in New Zealand” and is about submit his second on the phylodynamics of the hedgehog associated  <em>Salmonella</em> Enteritidis ST183.</p></li>
<li><p>Hugo and Sarah are based at PHF (formerly ESR) in Porirua and their projects are highly collaborative, working with multiple groups in PHF and David Welch at the University of Auckland and Alex Gavryushkin at the UNSW RNA Institute.</p></li>
<li><p>Sophie Yang (PhD student): Achieved notable milestones with her innovative regularised conditional clade distributions method, presented at international conference, manuscript near submission</p></li>
<li><p>Dr. Jonathan Klawitter (Postdoc - completed): Successfully completed his postdoctoral work with notable outputs including two published preprints on rogue taxa detection and credible set estimation methods during the reporting period.</p></li>
</ul>
<p><strong>New PhD students who started in this reporting period.</strong></p>
<ul>
<li><p>Eva (Yutong) Li (PhD student, newly enrolled 2025): Has developed a simulation pipeline for single-cell DNA sequencing variant calling to assess Bayesian phylogenetic inference and presented this work as a flash talk at the recent SMBE conference. The manuscript is in progress and will be completed soon. She is also implementing a simulator for calibrated coalescent point processing, which is currently undergoing validation within the LinguaPhylo framework. This work aims to study calibration-constrained diversification scenarios and to simulate data under a birth–death process for calibration-aware phylogenetic inference.</p></li>
<li><p>Yuan Xu (PhD student, newly enrolled): Has developed a Bayesian model averaging framework for comparing and integrating inferences across competing parametric population models using coalescent theory. Presented poster at SMBE annual meeting in Beijing (July 2025), attracting significant interest from international researchers. Has a manuscript under second round peer review and has initiated a new ambitious project on comprehensive Bayesian phylogenetic model averaging using Reversible Jump MCMC.</p></li>
<li><p>Reilly Haskins began his PhD study in March. He has been investigating methods to explain the behaviour of large language models in order to make them more trustworthy and reliable. This appointment directly supports MBIE’s goal of building advanced data science capability in New Zealand. Reilly is being trained in cutting-edge methodologies for model interpretability, contributing to a new generation of highly skilled AI researchers.</p></li>
</ul>
<h2 id="vision-mātauranga">Vision Mātauranga</h2>
<p>Co-PI Lara Greaves is working on several Māori and Indigenous data sovereignty projects, including directing activities in <em>Te Mana Raraunga</em>. She is also currently appointed to the <em>Global Indigenous Data Alliance</em> executive committee and planning a meeting on AI in Mexico in early 2026, which will bring together 50 Indigenous data sovereignty and AI experts. Lara is also working to re-engage Te Mana Raraunga, including making it a trust - to help with guidance in the AI space and resources.</p>
<p>One of our key workplans focusses on understanding the adaptive potential of hihi (stitchbird). Hihi are a taonga species and iwi Ngati Manuhiri are their kaitiaki. Our research outcomes are regularly communicated to Ngati Manuhiri and they are supportive of our work. </p>
<p>Our work on AI systems designed for research discovery reflect <strong>equitable access to knowledge</strong> and can support Māori researchers and communities. Future applications of claim verification pipelines to <strong>environmental and health sciences</strong> have direct relevance to Māori well-being and stewardship.</p>
<p>Piki Rātapu was funded through this grant for a <em>Ngā Motu Whakahī</em> summer scholarship, and is planning further studies in biology as a result. Eric Marshall, Tori Diamond, and Ben Barton (all former awardees) are all still continuing with their advanced studies, demonstrating the value of our summer scholarship offering.</p>
<h2 id="collaboration-and-partnerships-domestic-and-international-collaboration"><strong>Collaboration and Partnerships </strong>(Domestic and international collaboration)</h2>
<p><strong>Live and Transparent Data Science  </strong>We have built this year on our collaboration from Globus Labs (<a href="https://labs.globus.org/">https://labs.globus.org/</a> and <strong>Argonne National Labs, Chicago, USA</strong>).  This collaboration is focussed on capturing <strong>workflow provenance</strong> and the generation of provenance artifacts seamlessly without the user’s intervention, thus allowing us to create a complete and reproducible representation of analytical activity.  Both partners are contributing time and technical resources to the project.  As an outcome of this collaboration Ellerm (PhD student) presented at the industry Conference “Globus World” in Chicago, USA, as the featured closing presentation. Throughout the year we have also continued to build ties with the <strong>RO-Crate community—which is distributed throughout the EU</strong>, and Australia.</p>
<p>Partnering with <strong>Stencila</strong> (a New Zealand software company) our work on AI content provenance and GraphRAG directly supports New Zealand’s national data science priorities of trustworthy AI, openness, reproducibility, and responsible digital infrastructure. By enabling researchers to clearly distinguish human- from AI-generated content, we address an important transparency issue that aligns with NZ government guidance on trustworthy AI use. The GraphRAG and DocSQL initiatives further contribute by making scientific literature queryable as structured data, enabling more reliable AI assistance and supporting policy directions around open science, innovation, and data-driven decision-making.</p>
<p>The project benefited from an <strong>Astera Institute Open Science Fellowship</strong> (Dec 2023–Mar 2025), which provided resources and mentoring to advance Stencila’s AI integration. This fellowship complemented MBIE support and underscored the project’s international relevance. We also established a collaboration with Kùzu DB, whose technical guidance and feature development accelerated our GraphRAG and DocSQL work. No EU Horizon or NZ industry co-funding was received during this period.</p>
<p>Domestically, Stencila partnered with <strong>Dragonfly Data Science</strong>, whose technical expertise helped accelerate AI feature development. Internationally, the Astera Fellowship created a strong link with the open science community, while collaboration with Kùzu DB (Canada) advanced our technical capacity in graph-based retrieval.</p>
<p><strong>Trusted Data Science  </strong>We continue to partner with the <strong>Allen AI Institute (Based in Seattle</strong>), on a significant project called Nora. This extensive initiative involves multiple teams creating sub-agents designed to develop Artificial General Intelligence (AGI)-like algorithms. With our main collaborator, Niket Tandon, we are contributing to the scientific claim verification team, currently focusing on evaluating the claims in sets of research papers. This partnership is allowing us to access some of their state-of-the-art GenAI capabilities, experimental methods and compute infrastructure that is much larger and more performant than anything available currently in New Zealand. It also adds around 0.5 FTE to our project, funded by Allen AI.</p>
<p><strong>Biological Data Sciences   </strong>Hui Zhen Tan and Eirian Perkins presented their research to the Hihi Recovery Group as part of a Hihi Research Evening in February 2024. This led to discussions on the translocation history of Hihi and, following a literature review of all previous Hihi. This research utilises information from whole genome resequencing data and a Hihi genome assembly, <strong>funded by Genomics Aotearoa</strong>. We continue to work closely with the <strong>Department of Conservation</strong> Hihi Recovery group and our results have supported the development of management strategies, including translocations between current hihi populations to bolster genetic diversity.</p>
<p>We have strong connections to the <strong>High Quality Genomics and Population Genomics projects within Genomics Aotearoa</strong>, which Anna Santure co-leads.  Additionally Co-PI Paul Gardner is <strong>interacting with RNA Platform</strong> pillar leads Dr. Sarah Diermeier &amp; Dr. William Kelton to discuss bioinformatic method design challenges for the “target selection” and “payload design” challenges for RNA vaccines. We are collaborating with the <strong>RNA Platform</strong> to develop a pipeline for analysing pathogen genomes and identifying likely vaccine targets, in addition we are developing tools for optimising RNA sequences for vaccine applications.</p>
<p>For our population health focus, our main international collaborators on the salmonellosis project (Salmonella Enteritidis ST 183) are the <strong>German Federal Institute for Risk Assessment (BfR)</strong> and the <strong>Robert Koch Institute (also in Germany</strong>), and the <strong>Institute of Zoology, Zoological Society of London</strong>. They have provided genomic data and expertise on salmonellosis in Europe.  The <em>Neisseria meningitidis</em> project, based in New Zealand represent important collaborations between <strong>ESR, the Ministry for Primary Industries, the Ministry of Health</strong> and the food industry along with Massey University, the University of Auckland and the University of Canterbury, linking expertise in data science, genomics, phylodynamics, epidemiology and public health.  </p>
<p>This work has major implications for how two major diseases of public health concern are controlled. They will benefit public health and policy makers, and grow capability in genomic epidemiology in <strong>ESR</strong>, providing career opportunities for two excellent researchers.  </p>
<h2 id="support-for-nationally-significant-databases-and-collections">Support for Nationally Significant Databases and Collections</h2>
<ul>
<li><p>Development of <strong>curated research claim datasets</strong> that can be integrated with New Zealand’s broader AI/data infrastructures.</p></li>
<li><p>Contribution to open knowledge repositories that enable reuse of scientific findings, supporting national research capacity and transparency.</p></li>
<li><p>Our <em>Salmonella </em>and <em>Yersinia </em>studies have contributed to the genome sequencing databases.</p></li>
</ul>
